internal/wrphandlers/mocktr181/handler.go

```diff
@@ -173,32 +178,37 @@ func (h Handler) get(names []string) (int64, []byte, error) {
 					Value:      mockParameter.Value,
 					DataType:   mockParameter.DataType,
 					Attributes: mockParameter.Attributes,
+					Message:    "Success",
+					Count:      1,
```

Comment:
note: non-wildcard `get`s will always have a Count of 1 (based on `parodus2ccsp` and `wdmp-c`)